<?xml version="1.0"?>
<launch>
	<rosparam ns="color_range" command="load" file="$(find zeabus_vision)/params/color_down.yaml">
	</rosparam>
	<node pkg="zeabus_vision" name="color_range_down" type="color_range_down_new.py" output="screen">
		<param name="topic_down" value='/rightcam_bottom/image_raw/compressed'/>
	</node>
</launch>
